repo.yml

```diff
@@ -4,3 +4,5 @@ publishMetadata: true
 upstream:
   - repo: 'balena-sdk'
     url: 'https://github.com/balena-io/balena-sdk'
+  - repo: 'docker-qemu-transpose'
+    url: 'https://github.com/balena-io-modules/docker-qemu-transpose'
```

Comment:
@thgreasi, just to confirm, any upstream repo can be added to this file, right? Like I've done for `docker-qemu-transpose`.